packaging/src/docker/conf/hive-site.xml

```diff
@@ -60,4 +60,20 @@
         <name>metastore.metastore.event.db.notification.api.auth</name>
         <value>false</value>
     </property>
+    <property>
+        <name>iceberg.catalog</name>
```

Comment:
this seems redundant

Reply:
Added for debugging purposes, I remove it.